peridot.php

```diff
@@ -18,5 +18,7 @@
 
     $eventEmitter->on('code-coverage.start', function (AbstractCodeCoverageReporter $reporter) {
         $reporter->addDirectoryToWhitelist(__DIR__ . '/src');
+        // This cannot be covered, it is purely class aliases
+        $reporter->addFileToBlacklist(__DIR__ . '/src/shims.php');
```

Comment:
This functionality has been removed in sebastianbergmann/php-code-coverage@4712fc6b1328d62c5da00316e0c16d4f2147fed1

I am not going to attempt to correct for this.